src/ConstructiveSolidGeometry/SurfacePrimitives/TorusMantle.jl

```diff
@@ -40,6 +85,7 @@ get_φ_limits(tm::TorusMantle{T,Nothing}) where {T} = T(0), T(2π)
 
 get_θ_limits(tm::TorusMantle{T,<:Any,Tuple{T,T}}) where {T} = tm.θ[1], tm.θ[2]
 get_θ_limits(tm::TorusMantle{T,<:Any,Nothing}) where {T} = T(0), T(2π)
+get_θ_limits(tm::TorusMantle{T,<:Any,T}) where {T} = T(0), tm.θ
```

Comment:
The docs only allow for theta to be a tuple or nothing. The constructor does not specifically forbid using a real here, and the usage of a real might come naturally to the user, considering that this functionality exists for phi. Therefor I think it makes sense to add this function.

Reply:
When we introduced real `φ` back in #240, that was mostly changing how `φ` is saved in the `struct` during simulations. Config files are usually still written as `from`-`to` fields. I am not sure whether the user knows what would happen if they pass only a single value. But I also think it does not harm to have this functionality here.